Backend/lib/shared-services/src/IssueManagement/API/UI/Issue.hs

```diff
@@ -0,0 +1,31 @@
+module IssueManagement.API.UI.Issue where
+
+import qualified IssueManagement.Domain.Types.Issue.IssueReport as Domain
+import qualified IssueManagement.ProviderPlatform.Issue as Common
+import Kernel.Types.Id
+import Servant
+
+type IssueAPI =
+  Common.IssueCreateAPI
```

Comment:
Why these types are coming from Dashboard? Aren't service itself should have all the types?

Reply:
This is coming from within the library itself, however that module is only being used to reexport. We can remove it.